lib/network/modules/components/shared/Label.js

```diff
@@ -656,6 +628,50 @@ class Label {
 
 
   /**
+   * Get the current dimensions of the label
+   *
+   * @return {rect}
+   */
+  getSize() {
+    let lineMargin = 2;
+    let x = this.size.left;                 // default values which might be overridden below
+    let y = this.size.top - 0.5*lineMargin; // idem
+
+    if (this.isEdgeLabel) {
+      const x2 = -this.size.width * 0.5;
+
+      switch (this.fontOptions.align) {
+        case 'middle':
+          x = x2;
+          y = -this.size.height * 0.5
+          break;
+        case 'top':
+          x = x2;
+          y = -(this.size.height + lineMargin);
+          break;
+        case 'bottom':
+          x = x2;
+          y = lineMargin;
+          break;
+      }
+    }
+
+    var ret = {
+      left  : x,
+      top   : y,
+      width : this.size.width,
+      height: this.size.height,
+
+      // TODO: see if following required
```

Comment:
Is this meant to be addressed before merge?

Reply:
Sort of. I'll just address it now to be done with it.